src/managers/GuildMemberManager.js

```diff
@@ -432,6 +433,38 @@ class GuildMemberManager extends CachedManager {
     return this._add(data, cache);
   }
 
+  /**
+   * Adds a role to a member.
+   * @param {GuildMemberResolvable} user The user to add the role from
+   * @param {RoleResolvable} role The role to add
+   * @param {string} [reason] Reason for adding the role
+   * @returns {Promise<GuildMember|User|Snowflake>}
+   */
+  async addRole(user, role, reason) {
+    const userId = this.guild.members.resolveId(user);
+    const roleId = this.guild.roles.resolveId(role);
+
+    await this.client.api.guild(this.guild.id).members(userId).roles(roleId).put({ reason });
```

Comment:
Missing `s` in `guilds`:
```suggestion
    await this.client.api.guilds(this.guild.id).members(userId).roles(roleId).put({ reason });
```